lessons/lesson001.html

```diff
@@ -0,0 +1,51 @@
+<h1>HTML B�sico</h1>
```

Comment:
Por questões de semantica. Um bom titulo não é repetido ao longo do site.

Pensa nesse h1, se ele for mantido teremos o mesmo h1 em todas as paginas.

O que o pessoa costuma fazer é iniciar com o especifico, por exemplo.

```html
<h1> Aula 1  - HTML 101</h1>
```
Percebe como assim teremos titulos consistentes ao longo do documento?